src/components/Checkbox.tsx

```diff
@@ -79,7 +79,7 @@ export const CheckboxListItem: FunctionComponent<CheckboxListItemProps> = ({
   return (
     <>
       <div className="govuk-checkboxes__item">
-        <input id={`${id}Hidden`} type="hidden" name={name} value="false" />
+        {/*<input id={`${id}Hidden`} type="hidden" name={name} value="false" />*/}
         <input
           className="govuk-checkboxes__input"
           id={id}
```

Comment:
This hidden input with the same name as the visible input was resulting in form data looking like `[ "true", "false" ]` when a checkbox was checked.  I couldn't work out why the hidden input was there, and removing it still passes the tests and makes the value either `"true"` if checked or `"false"` if not.